package.json

```diff
@@ -3,9 +3,8 @@
   "version": "0.0.1",
   "description": "placeholder-app-description",
   "scripts": {
-    "postinstall": "npm run compile && npm run copy-artifacts && npm run build-app",
-    "copy-artifacts": "cp -R node_modules/@aragon/abis/os/artifacts/. artifacts/.",
```

Comment:
Do we not need this anymore in the latest version?

Reply:
No, we don't. What we might still need while we use truffle is a `TestImport.sol` if we import other artifacts that we want to use on the script hook. Like a Minime. Wonder if we can handle that better with a handler on the pluging internal logic 🤔